samples/apps/copilot-chat-app/scripts/Start.sh

```diff
@@ -4,16 +4,8 @@
 
 set -e
 
-# Parameters
-ClientId="$1"
-Tenant="common"
-AzureOpenAIOrOpenAIKey="${2:-}"
-
-ScriptDir="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
-cd "$ScriptDir"
```

Comment:
I think you might want to keep this -- what if someone tries to call this script from another directory? The subsequent calls will fail because they aren't found.